### DISCUSSION TITLE
Using newer mongo versions

### DISCUSSION THREAD

**Discussion by AlexGustafsson** [Yooooomi/your_spotify#472] [Q&A]
Has anyone tried running Your Spotify with mongo v7 or mongo v8? It works great with v6 as mentioned in the README.

**Comment by thedinz** [comment]
Cant soeak to newer version but the image i used said mongo 5+ but im using 4.4.0 and its working great.

**Comment by chiragkrishna** [comment]
i am using with
docker.io/mongo:8.2.6-noble@sha256:1a84ad76a5455fe55e49e95d2e5c5430710ad2fcf16fdd567df2481a1ca35892
